LICENSE-binary

```diff
@@ -323,10 +323,6 @@ org.apache.kerby:kerby-xdr:1.0.1
 org.apache.kerby:token-provider:1.0.1
 org.apache.yetus:audience-annotations:0.5.0
 org.apache.zookeeper:zookeeper:3.6.3
-org.codehaus.jackson:jackson-core-asl:1.9.13
-org.codehaus.jackson:jackson-jaxrs:1.9.13
-org.codehaus.jackson:jackson-mapper-asl:1.9.13
-org.codehaus.jackson:jackson-xc:1.9.13
```

Comment:
Since we are removing direct dependency, removing these entries. I believe we only provide direct dependencies here and not transitive dependencies. But please let me know if this is not the case.

Reply:
LICENSE-binary should include all the dependencies in the binary release. Now Jackson 1.9.13 is still used from Avro and the jar files exist in binary tarball, so we should keep it as is. Let's remove it when we upgrade Avro.